documentation/source/user_section/installation.rst

```diff
@@ -96,8 +96,125 @@ In the context of SCT, it can be used:
   virtual machine
 - <your reason here>
 
-See https://github.com/neuropoly/sct_docker for more information. We also provide a
-`tutorial to install SCT via Docker <https://github.com/neuropoly/spinalcordtoolbox/wiki/testing#run-docker-image>`_.
+Option 1: Without GUI
+=====================
+
+First, `install Docker <https://docs.docker.com/install/>`_. Then, follow the examples below to create an OS-specific SCT installation.
+
+
+Ubuntu-based installation
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+   # Start from the Terminal
+   docker pull ubuntu:16.04
+   # Launch interactive mode (command-line inside container)
+   docker run -it ubuntu
+   # Now, inside Docker container, install dependencies
+   apt-get update
+   yes | apt install git curl bzip2 libglib2.0-0 gcc
+   # Note for above: libglib2.0-0 is required by PyQt
+   # Install SCT
+   git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+   cd sct
+   yes | ./install_sct
+   export PATH="/sct/bin:${PATH}"
+   # Test SCT
+   sct_testing
+   # save the state of the container. Open a new Terminal and run:
+   docker ps -a  # list all containers
+   docker commit <CONTAINER_ID> <YOUR_NAME>/ubuntu:ubuntu16.04
+
+CentOS7-based installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+   # Start from the Terminal
+   docker pull centos:centos7
+   # Launch interactive mode (command-line inside container)
+   docker run -it centos:centos7
+   # Now, inside Docker container, install dependencies
+   yum install -y which gcc git curl
+   # Install SCT
+   git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+   cd sct
+   yes | ./install_sct
+   export PATH="/sct/bin:${PATH}"
+   # Test SCT
+   sct_testing
+   # save the state of the container. Open a new Terminal and run:
+   docker ps -a  # list all containers
+   docker commit <CONTAINER_ID> <YOUR_NAME>/centos:centos7
+
+
+Option 2: With GUI
+==================
+
+In order to run scripts with GUI you need to allow X11 redirection.
+First, save your Docker image:
+
+1. Open another Terminal
+2. List current docker images
+
+.. code:: bash
+
+   docker ps -a
+
+3. Save container as new image
+
+.. code:: bash
+
+   docker commit <CONTAINER_ID> <YOUR_NAME>/<DISTROS>:<VERSION>
+
+For OSX and Linux users
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Create an X11 server for handling display:
+
+1. Install XQuartz X11 server.
+2. Check ‘Allow connections from network clientsoption inXQuartz\`
+   settings.
+3. Quit and restart XQuartz.
+4. In XQuartz window xhost + 127.0.0.1
+5. In your other Terminal window, run:
+
+   -  On OSX:
+      ``docker run -e DISPLAY=host.docker.internal:0 -it <CONTAINER_ID>``
+   -  On Linux:
+      ``docker run -ti --rm -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix <CONTAINER_ID>``
+
+For windows users
+~~~~~~~~~~~~~~~~~
+
+| 1.Install Xming
+| 2.Connect to it using Xming/SSH:
+| Open a new CMD window and clone this repository:
+| ``git clone https://github.com/neuropoly/sct_docker.git``
```

Comment:
strange that this repos needs to be cloned for the Windows case, but not for the Linux/macOs case. I'm a bit suspicious about the whole procedure written here.

Reply:
It looks like these steps are quite similar to the ones written in the actual `sct_docker` README. (See step 4 under https://github.com/neuropoly/sct_docker#usage). I looked at the edit history of that README, and an older wording clarifies this:

```
   Run (double click) ``windows/sct-win.xlaunch`` found in this repository. If you are using docker toolbox then then run``windows/sct-win_docker_toolbox.xlaunch``
```

So, it looks like cloning `sct_docker` for Windows is needed only for [these launch scripts](https://github.com/neuropoly/sct_docker/tree/master/windows). 

They were added in https://github.com/neuropoly/sct_docker/pull/2 and https://github.com/neuropoly/sct_docker/pull/4, so @zougloub and/or @fperdigon might be the best people to ask regarding this procedure.

Reply:
If that's the case then maybe we can move those scripts inside SCT's repos?  We already have a folder `contrib/docker/`

Reply:
Good thinking. That gave me an idea, too. Using Sphinx's `:download:` role, I was able to make inline download links to these files to avoid having to deal with git at all.

![Screenshot from 2020-10-27 19-51-49](https://user-images.githubusercontent.com/16181459/97374121-f6807080-188d-11eb-83ea-2fd834072f80.png)

Reply:
brilliant!